Networking/Networking/Model/Order.swift

```diff
@@ -133,7 +133,7 @@ public struct Order: Decodable, GeneratedCopiable, GeneratedFakeable {
         let paymentMethodID = try container.decode(String.self, forKey: .paymentMethodID)
         let paymentMethodTitle = try container.decode(String.self, forKey: .paymentMethodTitle)
 
-        let items = try container.decode([OrderItem].self, forKey: .items)
+        let items = try? container.decode([OrderItem].self, forKey: .items)
```

Comment:
To avoid modifying the initializer and to not hide/shadow a possible parsing error, you could do 

```suggestion
        let items = try? container.decodeIfPresent([OrderItem].self, forKey: .items) ?? []
```